app/src/main/java/net/aiscope/gdd_app/model/MalariaSpecies.kt

```diff
@@ -0,0 +1,10 @@
+package net.aiscope.gdd_app.model
+
+@Suppress("MagicNumber") // these are IDs
+enum class MalariaSpecies(val id: Int) {
```

Comment:
Like having one 'Model.kt' file with all these classes you mean?

Reply:
I'd rather keeping each class in its file, but that's probably a result of my familiarity with Java... XD I think @jbors comment makes total sense, so I'm putting it all together again, but renaming the file to "Model.kt"...

Reply:
I file called `Model.kt` in the `.model` package? I think that we can do better :P 
How about just `Sample.kt`? (since `Sample` is the main domain model`

Reply:
SampleModels.kt? I agree Models.kt in the models package is redundant, but hey: We're using the Kotlin file with the same purpose of a package, aren't we? Sample.kt sounds off to me as well, as there is not only the Sample model in there... So, how about SampleModels.kt? Do you like the idea?

Reply:
Agreed with @eduardb having SampleModels.kt by dm. Thanks!!